### ISSUE TITLE
v1.1.16: java.lang.BootstrapMethodError: Exception from call site #10 bootstrap method

### ISSUE THREAD

**Issue by drewbitt** [hedzr/android-file-chooser#51]
```
java.lang.BootstrapMethodError: Exception from call site #10 bootstrap method
        at com.obsez.android.lib.filechooser.ChooserDialog.<clinit>(ChooserDialog.java:1529)
        at com.teamb.chzonk.ui.settings.SettingsFragment$PrefsFragment.onPreferenceTreeClick(SettingsFragment.kt:73)
        at androidx.preference.Preference.performClick(Preference.java:1173)
        at androidx.preference.Preference.performClick(Preference.java:1148)
        at androidx.preference.Preference$1.onClick(Preference.java:172)
        at android.view.View.performClick(View.java:6597)
        at android.view.View.performClickInternal(View.java:6574)
        at android.view.View.access$3100(View.java:778)
        at android.view.View$PerformClick.run(View.java:25885)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.ClassCastException: Bootstrap method returned null
        at com.obsez.android.lib.filechooser.ChooserDialog.<clinit>(ChooserDialog.java:1529) 
        at com.teamb.chzonk.ui.settings.SettingsFragment$PrefsFragment.onPreferenceTreeClick(SettingsFragment.kt:73) 
        at androidx.preference.Preference.performClick(Preference.java:1173) 
        at androidx.preference.Preference.performClick(Preference.java:1148) 
        at androidx.preference.Preference$1.onClick(Preference.java:172) 
        at android.view.View.performClick(View.java:6597) 
        at android.view.View.performClickInternal(View.java:6574) 
        at android.view.View.access$3100(View.java:778) 
        at android.view.View$PerformClick.run(View.java:25885) 
        at android.os.Handler.handleCallback(Handler.java:873) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
```

This is only present on 1.1.16. Tested: Android TV API 28, code for the dialog is

```
ChooserDialog(activity)
                    .withFilter(true, false)
                    .withStartFile(Environment.getExternalStorageDirectory().absolutePath)
                    .withResources(R.string.title_choose_folder, R.string.title_choose, R.string.dialog_cancel)
                    .withChosenListener { path, pathFile ->
                        Toast.makeText(activity, "FILE: $path / $pathFile", Toast.LENGTH_SHORT).show()
                    }
                    .build()
                    .show()
```

**Comment by Guiorgy** [comment]
Acording to [this](https://github.com/NordicSemiconductor/Android-BLE-Library/issues/38) guy, restarting andoid studio and the emilator fixed the issue? Have you tried this?

Also, shouldn't withChosenListener look more like this:
```java
.withChosenListener ( (path, pathFile) -> {
      Toast.makeText(activity, "FILE: $path / $pathFile", Toast.LENGTH_SHORT).show()
 })
```

**Comment by drewbitt** [comment]
On first restart no but then after cleaning a second time and restarting the emulator it's fixed for v1.1.16. Thanks.
